### PR TITLE
Chore: Support rules without documentation

### PR DESCRIPTION
**Pull request by qzhou1607**
## Pull request checklist

Make sure you:

- [ ] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [ ] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/contributing/pull-requests/#commitmessages)

### CODE REVIEW

File: src/server/routes/scanner.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ const ruleStatus = {
     warning: 'warning'
 };
 
+const noDocRules = ['optimize-image'];
```
Comment (alrra) [inline src/server/routes/scanner.js]: Isn't there a way to automatically determine this, so that we don't need to update this list manually?

Comment (sarvaje) [reply src/server/routes/scanner.js]: I'm not sure we can do that with the current version of sonarwhal

Comment (molant) [reply src/server/routes/scanner.js]: Also this should be only temporary and in a per case base. I'd rather us spend our time in other things 😊

Comment (alrra) [reply src/server/routes/scanner.js]: > Also this should be only temporary 

Ok.

**Pull request closed without merging**